samples/algorithms/QuantumHelloWorld.qs

```diff
@@ -0,0 +1,24 @@
+/// # Quantum Hello World!
+///
+/// This Q# code generates a random bit by setting a qubit in a superposition of the
+/// computational basis states |0〉 and |1〉, and returning the measurement result.
+namespace QuantumHelloWorld {
+    // Import the `Message` function and the `H` operation used in the program.
+    open Microsoft.Quantum.Intrinsic;
```

Comment:
Technically this import is no longer required, though it doesn't hurt. The Intrinsic namespace is now part of the default prelude.

Reply:
If it's no longer required, I think that it would be better to remove it.

Reply:
Removed the `open` statement.